### ISSUE TITLE
Error: app/R-3.5.1-win.exe failed to download.

### ISSUE THREAD

**Issue by lexizhu** [ficonsulting/RInno#112]
I've tried:
<create_app(
  app_name = "fapp", 
  app_dir = "app", 
  include_R = TRUE)> and
<create_app(
  app_name = "fapp", 
  app_dir = "app", 
  include_R = TRUE,
  R_version = '3.5.1')>

Both gave me : 
"Downloading R-3.5.1 ...https://cran.r-project.org/bin/windows/base/R-3.5.1-win.exe is not a valid URL.

This is likely to have happened because there was a change in the URL.

This might have already been fixed in the latest version of RInno. Install it with remotes::install_github('ficonsulting/RInno').

If this doesn't help please submit an issue: https://github.com/ficonsulting/RInno/issues

- Thanks!"

I've run remotes::install_github('ficonsulting/RInno') and still not work. Then I tried : remotes::install_github('ficonsulting/RInno', force = TRUE) and still got same Error message. 

I've searched a little bit but found no solution. Could you please help me with this?

Thanks a lot.

**Comment by Dripdrop12** [comment]
Can you download R from CRAN? I can't replicate this error.

**Comment by lexizhu** [comment]
Thanks so much for your reply. I can download R from CRAN and the URL 'https://cran.r-project.org/bin/windows/base/R-3.5.1-win.exe' actually worked very well when I pasted it into URL bar. Tried again today and same error here. I'll let my friends try and see if they will get the same error.  Thanks again.
![image](https://user-images.githubusercontent.com/30455157/50389721-a0f64c80-0768-11e9-913f-0693bbc9db3a.png)

**Comment by Dripdrop12** [comment]
I see what happened - R-3.5.2 was released on 12/20. 

There must have been a few days when 3.5.1 had not been archived yet. I checked the link and it is there now.

**Comment by Dripdrop12** [comment]
You can always manually download the file and place it in the `app_dir` directory. RInno should pick it up and use it instead of trying to download it.

**Comment by lexizhu** [comment]
Thanks again! Just some feedback.

It worked very well on my friends R-3.5.1 and R-3.5.2. I've updated R to 3.5.2, still got same error message. But after I manually pasted the R-3.5.2-win.exe to the directory, it worked really well.

Details attached as the following:
![image](https://user-images.githubusercontent.com/30455157/50533180-39247500-0b5f-11e9-8a38-ede2a59773c8.png)

Thanks again for this fantastic Package and the answers!
